### PR TITLE
Fix href link to Nextjs link

### PR DESCRIPTION
**Pull request by thuyvy3110**
Change all hash href link from normal link a to Nextjs Link #43

### CODE REVIEW

File: views/components/footer/GoTop.js [hunk 1]
```diff
@@ -2,11 +2,14 @@ import React from 'react'
 import styled from 'styled-components/macro'
 import Icon from '~/views/components/common/Icon'
 import { HOME } from '~/consts/pages'
+import Link from 'next/link'
 
 const GoTop = () => (
-  <Container href={`#${HOME}`}>
-    <GoTopIcon name="goTop" />
-  </Container>
+  <Link href={`#${HOME}`}>
```
Comment (DeKal) [inline views/components/footer/GoTop.js]: try `const Container = styled(Link)...

File: views/components/nav/NavItem.js [hunk 2]
```diff
@@ -2,22 +2,24 @@ import React from 'react'
...
-    <Container
-      data-test-id="nav-link"
-      href={`#${page}`}
-      onClick={() => {
-        setSelectedPage(page)
-      }}
-    >
-      <Translate id={page} />
-    </Container>
+    <Link href={`#${page}`}>
```
Comment (DeKal) [inline views/components/nav/NavItem.js]: same above

**Comment by DeKal** [comment]
The test case is failed because we trying to use Link while mock a click to redirect with Jest
https://github.com/zeit/next.js/issues/1827#issuecomment-306740374
see link for related solution

File: views/components/footer/GoTop.js (hunk 1 above)
Comment (DeKal) [reply views/components/footer/GoTop.js]: if the style is not good, we can consider  previous solution

**Comment by thuyvy3110** [comment]
Using Link Nextjs could not help the Scroll more smoothly, it's better to use Link to link two or more pages but in our case no useful here

**Pull request closed without merging**